logs.tf

```diff
@@ -0,0 +1,114 @@
+data "aws_iam_policy_document" "firehose_policy" {
+  count = local.is_kinesis_firehose_logs_enabled
+  statement {
+    sid = "SendStreamToBucket"
+    actions = [
+      "s3:AbortMultipartUpload",
+      "s3:GetBucketLocation",
+      "s3:GetObject",
+      "s3:ListBucket",
+      "s3:ListBucketMultipartUploads",
+      "s3:PutObject",
+      "s3:PutObjectAcl",
+    ]
+    resources = [
+      var.kinesis_firehose_log_bucket_arn,
+      "${var.kinesis_firehose_log_bucket_arn}/${var.kinesis_firehose_log_bucket_prefix}*",
+    ]
+  }
+  statement {
+    sid = "UseTheStream"
+    actions = [
+      "kinesis:DescribeStream",
+      "kinesis:GetShardIterator",
+      "kinesis:GetRecords",
+    ]
+    resources = [
+      aws_kinesis_firehose_delivery_stream.log_stream[0].arn,
+    ]
+  }
+  #statement {
```

Comment:
Do we need these?

Reply:
Not really, it was a copy from one of the projects we were consolidating.
I will remove the commented code.